worker.go

```diff
@@ -317,30 +282,30 @@ func go_frankenphp_worker_handle_request_start(threadIndex C.uintptr_t) C.bool {
 	return C.bool(true)
 }
 
-//export go_frankenphp_finish_request
-func go_frankenphp_finish_request(threadIndex C.uintptr_t, isWorkerRequest bool) {
+//export go_frankenphp_finish_worker_request
+func go_frankenphp_finish_worker_request(threadIndex C.uintptr_t) {
 	thread := phpThreads[threadIndex]
 	r := thread.getActiveRequest()
 	fc := r.Context().Value(contextKey).(*FrankenPHPContext)
 
-	if isWorkerRequest {
-		thread.workerRequest = nil
-	}
-
 	maybeCloseContext(fc)
+	thread.workerRequest = nil
+	thread.Unpin()
 
 	if c := fc.logger.Check(zapcore.DebugLevel, "request handling finished"); c != nil {
-		var fields []zap.Field
-		if isWorkerRequest {
-			fields = append(fields, zap.String("worker", fc.scriptFilename), zap.String("url", r.RequestURI))
-		} else {
-			fields = append(fields, zap.String("url", r.RequestURI))
-		}
-
-		c.Write(fields...)
+		c.Write(zap.String("worker", fc.scriptFilename), zap.String("url", r.RequestURI))
 	}
+}
 
-	if isWorkerRequest {
-		thread.Unpin()
+// when frankenphp_finish_request() is directly called from PHP
+//
+//export go_frankenphp_finish_request_manually
+func go_frankenphp_finish_request_manually(threadIndex C.uintptr_t) {
```

Comment:
```suggestion
func go_frankenphp_php_finish_request(threadIndex C.uintptr_t) {
```

?

I'm not fond of the name I suggest, but "manually" sounds a bit weird.

Reply:
Yeah I guess you're right 😅 . I changed it to `go_frankenphp_finish_php_request`